src/overlays/actors/ovl_En_Niw_Lady/z_en_niw_lady.c

```diff
@@ -20,43 +36,536 @@ const ActorInit En_Niw_Lady_InitVars = {
+    f32 frames;
+
+    if (Text_GetFaceReaction(globalCtx, 8) != 0) {
+        arg2 = 8;
+    }
+    if (arg2 != this->unk_270) {
+        this->unk_275 = 0;
+        this->unk_276 = 1;
+        this->unk_270 = arg2;
+        switch (arg2) {
+            case 10:
+                this->unk_275 = 1;
+            case 9:
+                frames = SkelAnime_GetFrameCount(&D_060007D0);
+                SkelAnime_ChangeAnim(&this->skelAnime, &D_060007D0, 1.0f, 0.0f, frames, 0, -10.0f);
+                break;
+            case 0:
+            case 1:
+            case 2:
+            case 3:
+            case 4:
+            case 5:
+            case 6:
+            case 8:
+            case 21:
+            case 22:
+            case 24:
+            case 29:
+                frames = SkelAnime_GetFrameCount(&D_06009F94);
+                SkelAnime_ChangeAnim(&this->skelAnime, &D_06009F94, 1.0f, 0.0f, frames, 0, -10.0f);
+                break;
+            case 7:
+            case 20:
+            case 23:
+            case 25:
+            case 26:
+            case 27:
+            case 28:
+                frames = SkelAnime_GetFrameCount(&D_06000718);
+                SkelAnime_ChangeAnim(&this->skelAnime, &D_06000718, 1.0f, 0.0f, frames, 0, -10.0f);
+                break;
+            case 100:
+                frames = SkelAnime_GetFrameCount(&D_0600A630);
+                SkelAnime_ChangeAnim(&this->skelAnime, &D_0600A630, 1.0f, 0.0f, frames, 0, -10.0f);
+                this->unk_276 = 0;
+                break;
+        }
+        return;
+    }
+}
+
+void func_80AB9F24(EnNiwLady* this, GlobalContext* globalCtx) {
+    f32 pad;
+    s32 pad1;
+    if (Object_IsLoaded(&globalCtx->objectCtx, this->ObjectAneIndex) &&
+        Object_IsLoaded(&globalCtx->objectCtx, this->ObjectOsAnimeIndex)) {
+        gSegments[6] = PHYSICAL_TO_VIRTUAL(globalCtx->objectCtx.status[this->ObjectAneIndex].segment);
+        SkelAnime_InitSV(globalCtx, &this->skelAnime, &D_060000F0, NULL, &this->limbDrawTable,
+                         &this->transitionDrawTable, 16);
+        gSegments[6] = PHYSICAL_TO_VIRTUAL(globalCtx->objectCtx.status[this->ObjectOsAnimeIndex].segment);
+        this->unk_27E = 1;
+        this->actor.gravity = -3.0f;
+        Actor_SetScale(&this->actor, 0.01f);
+        ActorShape_Init(&this->actor.shape, 0.0f, &ActorShadow_DrawFunc_Circle, 20.0f);
+        Collider_InitCylinder(globalCtx, &this->collider);
+        Collider_SetCylinder(globalCtx, &this->collider, &this->actor, &sCylinderInit);
+        this->unk_272 = 0;
+        this->actor.unk_1F = 6;
+        this->actor.draw = EnNiwLady_Draw;
+        switch (this->unk_278) {
+            case 0:
+                if (!(gSaveContext.itemGetInf[0] & 0x1000) && LINK_IS_CHILD) {
+                    pad = SkelAnime_GetFrameCount(&D_0600A630);
+                    SkelAnime_ChangeAnim(&this->skelAnime, &D_0600A630, 1.0f, 0.0f, (s16)pad, 0, 0.0f);
```

Comment:
everything looks good, id call this frameCount though since we want to reserve this name for actual stack padding